staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -81,6 +81,12 @@ type EvictionStrategy string
 // ---
 // +k8s:openapi-gen=true
 type VirtualMachineInstanceSpec struct {
+	// If specified, indicates the pod's priority.
+	// If not specified, the pod priority will be default or zero if there is no
+	// default.
+	// +optional
+	PriorityClassName string `json:"priorityClassName,omitempty"`
```

Comment:
since the api is being altered, we'll need to run code generation and commit the results "make generate"

Reply:
ack